src/data_collection/sentiment.py

```diff
@@ -1,15 +1,13 @@
 """
 Functionality for determining the sentiment of a given comment/string. This
-approach utilizes the Natural Language Toolkit in combination with
-SentiWordNet.
-
-This logic was informed by the following article written by "AI & Tech by Nidhika, PhD":
-https://medium.com/@nidhikayadav/sentiment-analysis-with-python-sentiwordnet-fd07ffc557
```

Comment:
I've moved this link to the project [wiki](https://github.com/AustinCullar/Astro/wiki/Sentiment-Analysis).